src/Tempest/Database/Migrations/MigrationManager.php

```diff
@@ -53,9 +54,12 @@ public function down(): void
     {
         try {
             $existingMigrations = Migration::all();
-        } catch (PDOException) {
-            // @todo should be handled better as PDO exception doesn't necessarily mean that the migrations table doesn't exist
-            event(new MigrationFailed('Migration', MigrationException::noTable()));
+        } catch (PDOException $exception) {
+            /** @throw UnhandledMatchError */
+            match ((string) $exception->getCode()) {
+                'HY000' => event(new MigrationFailed('Migration', MigrationException::noTable())),
```

Comment:
this is the code that the table doesn't exist. Writing this I thought about getting the table name from the exception itself 🤔